README.md

```diff
@@ -1,35 +1,466 @@
+
+### Sending Events ###
+
+```
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event");
+```
+
+When creating custom events, do not use the `swrve.*` or `Swrve.*` namespace for your own events. This is reserved for Swrve use only. Custom event names beginning with `Swrve.` are restricted and cannot be sent.
+
+An event payload can be added and sent with every event. This allows for more detailed reporting around events and funnels. The associated payload should be a dictionary of key/value pairs; it is restricted to string and integer keys and values. There is a maximum cardinality of 500 key-value pairs for this payload per event. This parameter is optional.
+
+```
+Dictionary<string,string> payload = new Dictionary<string,string>() {
+  {"key1", "value1"},
+  {"key2", "value2"}
+};
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event", payload);
+```
+
+### Send User Properties ###
+
+Assign user properties to send the status of the user. For example create a custom user property called `premium`, and then target non-premium users and premium users in the dashboard..
+
+```
+Dictionary<string, string> attributes = new Dictionary<string, string>();
+attributes.Add("premium", "true");
+attributes.Add("level", "12");
+attributes.Add("balance", "999");
+SwrveComponent.Instance.SDK.UserUpdate(attributes);
+```
+
+### Sending Virtual Economy Events ###
+
+If your app has a virtual economy, send the purchase event when users purchase in-app items with virtual currency.
+
+```
+string item = "some.item";
+string currency = "gold";
+int cost = 99;
+int quantity = 1;
+SwrveComponent.Instance.SDK.Purchase(item, currency, cost, quantity);
+```
+
+Send the currency given event when you give users virtual currency. Examples include initial currency balances, retention bonuses and level-complete rewards.
+
+```
+string givenCurrency = "gold";
+double givenAmount = 99;
+SwrveComponent.Instance.SDK.CurrencyGiven(givenCurrency, givenAmount);
+```
+
+To ensure virtual currency events are not ignored by the server, make sure the currency name configured in your app matches exactly the Currency Name you enter in the App Currencies section on the App Settings screen (including case-sensitive). If there is any difference, or if you haven’t added the currency in Swrve, the event will be ignored and return an error event called Swrve.error.invalid_currency. Additionally, the ignored events will not be included in your KPI reports. For more information, see [Add Your App](http://docs.swrve.com/getting-started/add-your-app/).
+
+### Sending IAP Events and IAP Validation ###
+
+If your app has in-app purchases, send the IAP event when a user purchases something with real money. The IAP event enables Swrve to build revenue reports for your app and to track the spending habits of your users.
```

Comment:
Add a prerequisites section.

Talk about a recommended plugin.  Talk about the need to setup sandbox purchases in iTunes connect.  Add a video?